integration-tests/devtools/src/test/java/io/quarkus/devtools/codestarts/quarkus/QuarkusCodestartRunIT.java

```diff
@@ -84,18 +85,24 @@ public void testGradleKotlinDSL(String language) throws Exception {
         generateProjectRunTests("gradle-kotlin-dsl", language, codestarts, Collections.emptyMap());
     }
 
+    /** Vert.x 4 Migration: resteasy reactive fixes */
+    @Disabled
     @ParameterizedTest
     @MethodSource("provideRunAloneExamples")
     public void testRunAloneCodestartsJava(String codestart) throws Exception {
         generateProjectRunTests("maven", "java", singletonList(codestart), Collections.emptyMap());
     }
 
+    /** Vert.x 4 Migration: resteasy reactive fixes */
+    @Disabled
```

Comment:
I actually fixed them.

Reply:
y. verifying. will clear and move to next